test/test_downloading.jl

```diff
@@ -15,10 +15,10 @@ end
 
         @info "Testing that we can download $(typeof(dataset)) data..."
 
-        variables = dataset isa ECCO4Monthly ? keys(ClimaOcean.ECCO.ECCO4_short_names) :
-                    dataset isa ECCO2Monthly ? keys(ClimaOcean.ECCO.ECCO2_short_names) :
-                    dataset isa ECCO2Daily ? keys(ClimaOcean.ECCO.ECCO2_short_names) :
-                    dataset isa EN4Monthly ? keys(ClimaOcean.EN4.EN4_short_names) :
+        variables = dataset isa ECCO4Monthly ? keys(ClimaOcean.ECCO.ECCO4_dataset_variable_names) :
+                    dataset isa ECCO2Monthly ? keys(ClimaOcean.ECCO.ECCO2_dataset_variable_names) :
+                    dataset isa ECCO2Daily ? keys(ClimaOcean.ECCO.ECCO2_dataset_variable_names) :
+                    dataset isa EN4Monthly ? keys(ClimaOcean.EN4.EN4_dataset_variable_names) :
```

Comment:
add the Copernicus test?

Reply:
I will do that once we have solid limited area support. Also it may be good to split it up because copernicus is a bit slow